sklearn/multiclass.py

```diff
@@ -282,6 +285,12 @@ class OneVsRestClassifier(
     array([2, 0, 1])
     """
 
+    _parameter_constraints = {
+        "estimator": [bool],
```

Comment:
We can handle this similar to `OutputCodeClassifier`:
https://github.com/scikit-learn/scikit-learn/blob/043ad6937d01965ea3722e175de42e17f0fb234f/sklearn/multiclass.py#L942-L945

```suggestion
        "estimator": [
            HasMethods(["fit", "decision_function"]),
            HasMethods(["fit", "predict_proba"]),
        ],
```

Reply:
@Micky774 actually the docstring is misleading, because this can be used also with regressors which might not have `decision_function` or `proba`. 
So the estimator validation should just be `HasMethods("fit")`

Reply:
Thanks for the input @francesco-tuveri